test/client/client.go

```diff
@@ -120,7 +120,7 @@ func (c RestClient) GetBridges(destAddr string, offset, limit uint) ([]*pb.Depos
 	if err != nil {
 		return nil, 0, err
 	}
-	bodyBytes, err := ioutil.ReadAll(resp.Body)
+	bodyBytes, err := io.ReadAll(resp.Body)
```

Comment:
What is the difference?

Reply:
`golint` suggests it, looks like `ioutil` is deprecated